src/less/components/search/index.less

```diff
@@ -52,6 +164,45 @@ hx-search {
   height: 2rem;
 }
 
+// ----- PRISTINE ----------
+hx-search-control > hx-search {
+  .SearchFacade(pristine);
+}
```

Comment:
This block of styles may be unnecessary.

Reply:
Block removed.